### PR TITLE
Notifications support

### PR DESCRIPTION
**Pull request by odinuv**
refactor: make postJobResult and patchJob return Job instead of array
feat: add getStartTime and getEndTime methods

### CODE REVIEW

File: src/Client.php [hunk 1]
```diff
@@ -179,7 +180,7 @@ public function updateJob(JobInterface $newJob): array
         return $this->sendRequest($request);
     }
 
-    public function patchJob(string $jobId, JobPatchData $patchData): array
+    public function patchJob(string $jobId, JobPatchData $patchData): JobInterface
```
Comment (MiroCillik) [inline src/Client.php:183]: Super, ze si toto poriesil tiez

**Review by MiroCillik — APPROVED** [review]
Pride mi to OK, len to bude musiet byt nova major verzia, kedze sa zmenili ten return type z array na JobInterface

File: src/Client.php (hunk 1 above)
Comment (odinuv) [reply src/Client.php:183]: Ono to bylo nutný kvůli tomuhle https://github.com/keboola/job-queue-daemon/pull/174/files#diff-5c2f13b95cb1b0bc375346d400bd208f08c9468ae829d6c922a76ca2db47e3fcR96

**Comment by odinuv** [comment]
Jojo, major verze bude

**Pull request merged**